test/data/manifest-checksums.txt

```diff
@@ -39,7 +39,7 @@ f3a32b59496828a50a38db2e19cd594b6783702a  centos_10-x86_64-wsl-empty_rhel.json
 3de7877c3793c9da3016819659e400ee42a4adb8  centos_9-aarch64-ami-partitioning_dos_lvm.json
 f5ff587ed1e4ae106a473aa48df8aebc135446b5  centos_9-aarch64-ami-partitioning_lvm.json
 eafc0c15018bcd841e3257742f181a64a3dbf706  centos_9-aarch64-ami-partitioning_plain.json
-f5f0fd4917c3061673985a211b9d02ca3173f992  centos_9-aarch64-edge_ami-edge_ostree_pull_user.json
```

Comment:
Nice to have the manifest diffs that show that this actually has the desired effects \o/